tests/unit/install.js

```diff
@@ -304,7 +306,7 @@ describe('Installation flow', () => {
 
     // https://github.com/medic/horticulturalist/issues/39
     it('viewQueries should ignore mango indexes');
-    it('we should also warm mango indexes')
+    it('we should also warm mango indexes');
```

Comment:
This should have been caught by eslint. I checked and I don't think our eslint config is actually running - I badly broke a rule and it didn't complain. I think the `package.json` `scripts/eslint` needs to be updated to:

```
"eslint": "eslint \"src/**/*.js\" \"tests/**/*.js\"",
```

After doing that it _still_ didn't fail on the missing semi because our .eslintrc rules differ from medic/medic. Once I copied over this rule it started failing:

```
    "semi": [
      "error",
      "always"
    ],
```

Can you add those changes to this PR and also consider copying more eslint rules over? We should probably look at "publishing" a medic standard so we don't have to copy and paste into every repo, but that can wait for another PR...

Reply:
Done. I copied all rules (that made sense) from `medic/medic`.